overlord/configstate/config/helpers.go

```diff
@@ -281,3 +282,24 @@ func DeleteSnapConfig(st *state.State, snapName string) error {
 	}
 	return nil
 }
+
+func featureKey(flag features.SnapdFeature) string {
+	return "experimental." + flag.String()
+}
+
+// GetFeatureFlag returns the value of a given feature flag.
+func GetFeatureFlag(tr Conf, flag features.SnapdFeature) (bool, error) {
```

Comment:
does this have its own tests?

Reply:
I'll check, don't recall from the top of my head.

Reply:
It doesn't, I'll add some next.